server/controllers/api/channel/claims/getChannelClaims.js

```diff
@@ -3,30 +3,37 @@ const chainquery = require('chainquery').default;
 const getClaimData = require('server/utils/getClaimData');
 const { returnPaginatedChannelClaims } = require('./channelPagination.js');
 
-const getChannelClaims = async (channelName, channelShortId, page) => {
-  const channelId = await chainquery.claim.queries.getLongClaimId(channelName, channelShortId);
-  const params = { content_type: [
-    'image/jpeg',
-    'image/jpg',
-    'image/png',
-    'image/gif',
-    'video/mp4',
-  ] };
+const getChannelClaims = async (channelName, channelLongId, page) => {
+  const params = {
+    content_type: ['image/jpeg', 'image/jpg', 'image/png', 'image/gif', 'video/mp4'],
+  };
+
+  let channelShortId = await chainquery.claim.queries.getShortClaimIdFromLongClaimId(
```

Comment:
Line 11 is the main fix, after changing input params from expecting shortId to expecting longId.